README.md

```diff
@@ -1,2 +1,224 @@
-# sdk-python
+# Vela Python SDK
+[![license](https://img.shields.io/crates/l/gl.svg)](LICENSE)
+
 Python SDK for Vela (Target's official Pipeline Automation Framework)
+
+> Vela is in active development and is a pre-release product.  Please use it at your own risk in production.
+
+This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
+
+- API version: 0.4.3
```

Comment:
Thanks for the contribution 🎉 

I noticed you made a commit for updating some things for the `0.6.1` API.

Should this be updated to `0.6.1` as well?

Reply:
Opps, missed that one.  I didn't let the new generation overwrite `README.md` so I could keep my changes.